man/man1/fzf.1

```diff
@@ -98,62 +98,62 @@ Fuzzy matching algorithm (default: v2)
 .br
 
 .TP
-.BI "-n, --nth=" "N[,..]"
+.BI "\-n, \-\-nth=" "N[,..]"
 Comma-separated list of field index expressions for limiting search scope.
 See \fBFIELD INDEX EXPRESSION\fR for the details. When you use this option with
 \fB--with-nth\fR, the field index expressions are calculated against the
 transformed lines (unlike in \fB--preview\fR where fields are extracted from
```

Comment:
```suggestion
\fB\-\-with\-nth\fR, the field index expressions are calculated against the
transformed lines (unlike in \fB\-\-preview\fR where fields are extracted from
```

Reply:
Ah, thanks. I think i need to double-check my changes again.